clustered/client/src/test/java/org/ehcache/clustered/client/ClusteredCacheDestroyTest.java

```diff
@@ -107,6 +107,16 @@ public void testDestroyFreesUpTheAllocatedResource() throws CachePersistenceExce
   }
 
   @Test
+  public void testDestroyUnknownCacheAlias() throws Exception {
+    clusteredCacheManagerBuilder.build(true).close();
+
+    PersistentCacheManager cacheManager = newCacheManagerBuilder().with(cluster(CLUSTER_URI).expecting()).build(true);
+
+    // No exception thrown here indicates it succeeded as other test show
+    cacheManager.destroyCache("clustered-cache");
```

Comment:
Not without doing some of the "nastiness" resorted to in `DefaultClusteringServiceTest` with the aid of `ObservableEhcacheServerEntityService`.